integration/setup/common.go

```diff
@@ -44,43 +45,31 @@ func common(ctx context.Context, config Config) error {
 	}
 
 	{
-		c := chartvalues.CertOperatorConfig{
-			CommonDomain:       env.CommonDomain(),
-			RegistryPullSecret: env.RegistryPullSecret(),
-			Vault: chartvalues.CertOperatorVault{
-				Token: env.VaultToken(),
-			},
-		}
-
-		values, err := chartvalues.NewCertOperator(c)
-		if err != nil {
-			return microerror.Mask(err)
-		}
-
-		err = config.Release.InstallOperator(ctx, key.CertOperatorReleaseName(), release.NewStableVersion(), values, corev1alpha1.NewCertConfigCRD())
+		_, err := config.K8sClients.K8sClient().CoreV1().Secrets("giantswarm").Create(credentialDefault())
 		if err != nil {
 			return microerror.Mask(err)
 		}
 	}
 
 	{
-		err := ensureCertConfigsInstalled(ctx, env.ClusterID(), config)
+		err := installCertOperator(ctx, config)
 		if err != nil {
 			return microerror.Mask(err)
 		}
 	}
 
 	{
-		c := chartvalues.NodeOperatorConfig{
-			RegistryPullSecret: env.RegistryPullSecret(),
-		}
-
-		values, err := chartvalues.NewNodeOperator(c)
+		config.Logger.LogCtx(ctx, "level", "debug", "message", "ensuring drainerconfig CRD exists")
```

Comment:
The `drainerconfig` CRD was created by node-operator, but not anymore. We need to ensure it exists.